### PR TITLE
changing @yy to @UserName

### LINKED ISSUE #193: Create the Header component
**Description**
With the `Header` component having been built in Vocabulary with the markup and styling already defined, it's time to get the Vue version of the component.

The Header component is a pattern and as such has a lot of markup involved for even the very simple usecases. Having a Vue component with `slot`s takes a huge load off the developer to focus on the main content. 

References:
- creativecommons/vocabulary#145
- [Header @ Figma](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=179%3A0)

### PR DESCRIPTION
**Pull request by achilep**
**Fixes**
Fixes #193 by @kgodey 

**Description**
the Pr is to avoid sending the notification to Mr yy .

**Type of PR** 
This PR is a [refactor].

**Technicalities**
changing the @yy by @UserName

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 **Fixes**
 <!-- If PR only partly solves the issue, replace 'Fixes' below with 'Partially addresses' -->
-Fixes #XX by @YY
+Fixes #XX by @YourUserName
```
Comment (kgodey) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would just leave the `@` out altogether and update the instructions above to include the `@` sign. Also, the username should be of the person who created the issue, not the person who is creating the PR.

**Review by kgodey — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (achilep) [reply .github/PULL_REQUEST_TEMPLATE.md]: can i change it to  this @UserName

Comment (achilep) [reply .github/PULL_REQUEST_TEMPLATE.md]: thank @kgodey

Comment (kgodey) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think @username would be fine.

Comment (achilep) [reply .github/PULL_REQUEST_TEMPLATE.md]: i have done @kgodey

**Review by dhruvkb — APPROVED** [review]

**Review by kgodey — APPROVED** [review]

**Pull request merged**